getting_started/topics/secure-jboss-app/install-client-adapter.adoc

```diff
@@ -20,13 +20,23 @@ endif::[]
 
 . Change to the root directory of {appserver_name}.
 
-. Unzip the downloaded client adapter in this directory.
+. Unzip the downloaded client adapter in this directory.  For example:
 +
 [source,bash,subs=+attributes]
 ----
 $ unzip <filename>.zip
 ----
 
+ifeval::[{project_product}==true]
+. Return to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=core.service.rhsso[Red Hat customer portal].
+
+. Click the `Patches` tab.
+
+. Download the {project_version} patch for *Client Adapter for EAP 7*.
+
+. Unzip the ZIP file for this patch.
+endif::[]
+
```

Comment:
@andymunro This block is for me quite confusing, why do you propose the patch installation instead the latest version installation?